src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions.Compatibility/Extensions/IndentedStringBuilder.cs

```diff
@@ -25,6 +25,55 @@ namespace Uno.Extensions
 	/// </summary>
 	internal sealed class IndentedStringBuilder : IIndentedStringBuilder
 	{
+		// https://www.meziantou.net/split-a-string-into-lines-without-allocation.htm
+		// Must be a ref struct as it contains a ReadOnlySpan<char>
+		private ref struct LineSplitEnumerator
```

Comment:
Looks like it's very useful! It would make sense to put it in the Uno.Extensions folder: https://github.com/unoplatform/uno/tree/master/src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions